graphene_django/static/graphene_django/graphiql.js

```diff
@@ -108,7 +120,7 @@
   var activeSubscription = null;
 
   // Define a GraphQL fetcher that can intelligently route queries based on the operation type.
-  function graphQLFetcher(graphQLParams) {
+  function graphQLFetcher(graphQLParams, opts = { headers: {} }) {
```

Comment:
no need for default params here, if the `httpClient` function does all validation anyway